shenyu-client/shenyu-client-dubbo/shenyu-client-alibaba-dubbo/src/main/java/org/apache/shenyu/client/alibaba/dubbo/AlibabaDubboServiceBeanListener.java

```diff
@@ -100,19 +101,19 @@ private void handler(final ServiceBean<?> serviceBean) {
         }
         Method[] methods = ReflectionUtils.getUniqueDeclaredMethods(clazz);
         for (Method method : methods) {
-            ShenyuDubboClient shenyuDubboClient = method.getAnnotation(ShenyuDubboClient.class);
-            if (Objects.nonNull(shenyuDubboClient)) {
-                publisher.publishEvent(buildMetaDataDTO(serviceBean, shenyuDubboClient, method));
+            SoulDubboClient dubboClient = AnnotatedElementUtils.findMergedAnnotation(method, SoulDubboClient.class);
+            if (Objects.nonNull(dubboClient)) {
+                publisher.publishEvent(buildMetaDataDTO(serviceBean, dubboClient, method));
             }
         }
     }
 
-    private MetaDataRegisterDTO buildMetaDataDTO(final ServiceBean<?> serviceBean, final ShenyuDubboClient shenyuDubboClient, final Method method) {
+    private MetaDataRegisterDTO buildMetaDataDTO(final ServiceBean<?> serviceBean, final SoulDubboClient dubboClient, final Method method) {
```

Comment:
why is sloudubbo client

Reply:
@ShenyuDubboClient is a derived annotation of @SoulDubboClient.
```java
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.TYPE, ElementType.METHOD})
@Inherited
@SoulDubboClient(path = "")
public @interface ShenyuDubboClient {

    @AliasFor(annotation = SoulDubboClient.class)
    String path();

    @AliasFor(annotation = SoulDubboClient.class)
    String ruleName() default "";

    @AliasFor(annotation = SoulDubboClient.class)
    String desc() default "";

    @AliasFor(annotation = SoulDubboClient.class)
    boolean enabled() default true;
}
```
see `org.springframework.core.annotation.AliasFor` and `org.springframework.core.annotation.AnnotatedElementUtils#findMergedAnnotation`.

Reply:
Is it for compatibility? soul version, we are not going to maintain

Reply:
Yes, just for compatibility. So be it, close this PR ?

Reply:
@dragon-zhang thank you for pr， you already add my wechat ：xixy199195?   ,you can close this PR~